### PR TITLE
Update for the AWS SDK v2

### PR DESCRIPTION
**Pull request by Benjamin-deVries**
Upgrades aws-sdk to use the new amazon library.

### CODE REVIEW

File: ui.apps/pom.xml [hunk 1]
```diff
@@ -56,57 +49,12 @@
                 <artifactId>filevault-package-maven-plugin</artifactId>
                 <extensions>true</extensions>
                 <configuration>
-                    <group>aws-sdk</group>
+                    <group>jersey-bundle</group>
```
Comment (wrandall22) [inline ui.apps/pom.xml]: This should not have changed.

File: ui.apps/pom.xml [hunk 2]
```diff
@@ -56,57 +49,12 @@
...
-                            <artifactId>netty-codec</artifactId>
-                            <target>/apps/aws-sdk/install</target>
-                        </embedded>
-                        <embedded>
-                            <groupId>io.netty</groupId>
-                            <artifactId>netty-common</artifactId>
-                            <target>/apps/aws-sdk/install</target>
+                            <groupId>org.cru</groupId>
+                            <artifactId>jersey-bundle.core</artifactId>
+                            <target>/apps/jersey-bundle/install</target>
```
Comment (wrandall22) [inline ui.apps/pom.xml:57]: This should not have changed.

File: ui.apps/src/main/content/META-INF/vault/filter.xml [hunk 3]
```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <workspaceFilter version="1.0">
     <filter root="/apps/aws-sdk"/>
-</workspaceFilter>
+</workspaceFilter>
```
Comment (wrandall22) [inline ui.apps/src/main/content/META-INF/vault/filter.xml]: Let's add the missing newlines back to these various files.

File: META-INF/maven/org.cru/aws-sdk/pom.xml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>cru-awssdk-osgi</Bundle-SymbolicName>
+                        <Bundle-Description>This bundle wraps the AwsSdk</Bundle-Description>
+                        <Import-Package>
+
+                        </Import-Package>
+                        <Export-Package>
+                            software.amazon.awssdk.cloudsearch.*,
```
Comment (wrandall22) [inline META-INF/maven/org.cru/aws-sdk/pom.xml:32]: We do not only use cloudsearch from this SDK.

**Review by wrandall22 — CHANGES_REQUESTED** [review]
You'll want to update the project to use Java 11 instead of Java 8, as well as adding a `ui.apps.structure` module following suit with the other repos. You may want to consider updating `master` to `main` as part of this as well, at some point, to keep consistency.

File: META-INF/maven/org.cru/aws-sdk/pom.xml (hunk 4 above)
Comment (Benjamin-deVries) [reply META-INF/maven/org.cru/aws-sdk/pom.xml:32]: aah, yes. CloudSearch, CloudSearchDomain, and ssm. Any i'm missing?

Comment (wrandall22) [reply META-INF/maven/org.cru/aws-sdk/pom.xml:32]: Auth, client, and region calls at least.

Comment (Benjamin-deVries) [reply META-INF/maven/org.cru/aws-sdk/pom.xml:32]: I think we get those automatically, see MANIFEST.MF
Also: https://github.com/CruGlobal/aem-cloudsearch/pull/47
And: https://github.com/CruGlobal/cru-aem-commons/pull/317

File: META-INF/MANIFEST.MF
Comment (wrandall22) [inline META-INF/MANIFEST.MF:1]: Typically, these files are generated in the target directory using the maven plugin, vs putting it in here manually.

File: META-INF/maven/org.cru/aws-sdk/pom.xml (hunk 4 above)
Comment (wrandall22) [reply META-INF/maven/org.cru/aws-sdk/pom.xml:32]: There are no exports that I see in the `MANIFEST.MF` here in the PR.

File: pom.xml [hunk 5]
```diff
@@ -480,149 +469,18 @@
...
-                <scope>provided</scope>
-            </dependency>
-
-            <!-- Testing -->
-            <dependency>
-                <groupId>org.junit</groupId>
-                <artifactId>junit-bom</artifactId>
-                <version>5.4.1</version>
+                <groupId>org.cru</groupId>
+                <artifactId>aem-bom</artifactId>
```
Comment (wrandall22) [inline pom.xml:479]: Are we using anything from this BoM?

File: core/pom.xml [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>cru-awssdk-osgi</Bundle-SymbolicName>
+                        <Bundle-Description>This bundle wraps the AwsSdk</Bundle-Description>
+                        <Import-Package>
+
+                        </Import-Package>
```
Comment (wrandall22) [inline core/pom.xml:30]: If you mean to import nothing, you should probably specify `!*` here.

**Comment by Benjamin-deVries** [comment]
Closed in favor of: https://github.com/CruGlobal/aws-sdk/pull/9

**Pull request closed without merging**